plugin/inject-solr-plugin/src/main/java/org/hrodberaht/injection/plugin/junit/solr/SolrAssertions.java

```diff
@@ -74,6 +74,12 @@ public void cleanDataFromCollection() throws IOException, SolrServerException {
     }
 
 
+    public void cleanDataFromCollection(String query) throws IOException, SolrServerException {
```

Comment:
this is not needed within the framework, just use a wrapper class and call these two methods

Reply:
@alero the processDelete() as well as processCommit both are private methods and thus cannot be called from wrapper class too!!